src/client.ts

```diff
@@ -39,7 +39,7 @@ export const DEFAULT_CLIENT_CONFIG: Omit<ClientConfiguration, "secret"> = {
   client_timeout_buffer_ms: 5000,
   endpoint: endpoints.default,
   format: "tagged",
-  http2_session_idle_ms: 500,
+  http2_session_idle_ms: 5000,
```

Comment:
Are these idle timeout changes related to GOWAY?

Reply:
Yes I found the short idle timeout could trigger the race condition.